### DISCUSSION TITLE
[question] Working with the CI email verification rules

### DISCUSSION THREAD

**Discussion by Oloremo** [semantic-release/semantic-release#2508] [Configuration]
## Current behavior

We used semantic-release in Gitlab for quite some time with the default email of semantic-release-bot@martynus.net.

When we realized it was wrong and added GIT_AUTHOR_EMAIL to change the commit email to the expected email.

When we enabled the Gitlab feature 
```
Reject unverified users
Users can only push commits to this repository that were committed with one of their own verified emails.
```
And release fails with that option enabled with the error:
```
GitLab: You cannot push commits for 'semantic-release-bot@martynus.net'. You can only push commits that were committed with one of your own verified emails.
```

We checked and the release commits seem to be using the correct email:
(domain name changed to `example`)
```
commit eb81426791558036b8aa71a834374581d200f351 (origin/master)
Author: semantic-release-bot <devops-releaser@example.com>
Date:   Thu Mar 3 16:47:03 2022 +0000

    chore(release): 4.497.2
```

Not 100% sure why it mentions the old email here.

## Environment

- **semantic-release** version:  semantic-release@17.4.3
- CI environment: Gitlab
- Plugins used:
```
@semantic-release/gitlab@6.1.0
@semantic-release/changelog@5.0.1
@semantic-release/git@9.0.0'
```

**Comment by airtonix** [comment]
@Oloremo because it looks like you also need to override the other two ENVARS: https://semantic-release.gitbook.io/semantic-release/usage/configuration#git-environment-variables

**Reply by kiranpatel11** [reply]
For the email address of the bot users in Gitlab.
https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html#bot-users-for-projects
